README.rst

```diff
@@ -70,3 +70,41 @@ To generate correct files, please provide the following input to Cookiecutter:
                                  super project of this module, e.g. Invenio
                                  itself.
 ================================ =============================================
+
+Usage example
+--------------
```

Comment:
Oh, you mean as in 'the number of chars in the underlining the exact same number as the header'?